bin/augur

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import uuid
+import pprint
+import click
+import augur
+
+
+@click.command()
+@click.argument('config', type=str)
+@click.option('-v', '--verbose', is_flag=True)
+def run(config, verbose):
+    """Here comes a Roman guy with a spade in his hand.
```

Comment:
I like the funny message, but this one will appear in the help string for the CLI tool so it should probably have an actually helpful thing in it.